### PR TITLE
Учебный проект: холст

### PR DESCRIPTION
**Pull request by KuryakovDen**
---
:mortar_board: [Учебный проект: холст](https://up.htmlacademy.ru/javascript/18/user/42634/tasks/3)

:boom: https://htmlacademy-javascript.github.io/42634-code-and-magick-18/2/

### CODE REVIEW

File: js/stat.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  var maxTime = getMaxArrayValue(times);
+
+  for (var i = 0; i < players.length; i++) {
+    //   MAX_BAR      BAR[I]
+    // ----------- = --------
+    //  BAR_WIDTH       X
+
+    //  X = (BAR_WIDTH * BAR[I]) / MAX_BAR
+
+    ctx.fillText(players[i], START_GAP + (TEXT_GAP * i), textHeight); // Info about player's names
```
Comment (uladzimir-miadzinski) [inline js/stat.js]: you should use and prefer named functions instead code with comment

for example for these lines you can have functions like:
`setPlayerName`, `createBarColumn`, `setPlayerPoints`, `colorize` , or any another that you want and remove comments

Comment (KuryakovDen) [reply js/stat.js]: I have refactored my code and pushed my new pullrequest

Comment (KuryakovDen) [reply js/stat.js]: I have refactored my code again

**Comment by KuryakovDen** [comment]
I have refactored my code again

**Pull request merged**